storybookjs/storybook
# Addon-docs: Fix newline handling in ArgsTable code blocks

Issue: #11358 (propTypes) and #12848 (interface)

## What I did

Added `white-space: pre-wrap;` on code blocks (specifically). This does not affect inline code.

### Before:
![image](https://user-images.githubusercontent.com/8504538/96990345-462a0980-1527-11eb-8f18-9647dc5db2b5.png)

### After:
![image](https://user-images.githubusercontent.com/8504538/96990388-5510bc00-1527-11eb-92da-be2327cc2605.png)

## How to test

- Is this testable with Jest or Chromatic screenshots?
  - **No, unless ArgsTables are included in the screenshots**
- Does this need a new example in the kitchen sink apps?
  - **Yes**
  - **Added a propTypes example in `cra-kitchen-sink\src\components\Container.js` together with an ArgsTable in the test story: `cra-kitchen-sink\src\stories\test.stories.mdx`.**
  - **Added an interface example in `cra-ts-kitchen-sink\src\components\Button.tsx`.**
- Does this need an update to the documentation?
  - **No**

If your answer is yes to any of these, please make sure to include it in your PR.
